src/views/__tests__/HomeView.spec.ts

```diff
@@ -46,12 +46,7 @@ describe("HomeView.vue", () => {
         };
       },
       computed: {
-        selectedLangs: (): Record<TLangs, Languages> => {
-          return {
-            targetLang: Languages.EN,
-            sourceLang: Languages.UK,
-          };
-        },
+        selectedLangs: () => selectLanguagesModule.selectedLanguages,
```

Comment:
remove 
```
computed: {
        selectedLangs: (): Record<TLangs, Languages> => {
          return {
            targetLang: Languages.EN,
            sourceLang: Languages.UK,
          };
        },
        selectedLangs: () => selectLanguagesModule.selectedLanguages,
```
there is no need in it. When your component mounts it computes `selectedLangs` by itself. It calls `selectLanguagesModule.selectedLanguages` action, takes value from there and save it in the property `selectedLangs`. It is done when you call `mount` or `shallowMount`